modules/core/04-channel/keeper/multihop_handshake_test.go

```diff
@@ -246,12 +328,25 @@ func (suite *MultihopTestSuite) TestChanOpenAckMultihop() {
 			suite.SetupTest() // reset
 			tc.malleate()     // call ChanOpenInit and setup port capabilities
 
-			proof := suite.Z().QueryChannelProof(suite.Z().Chain.LastHeader.GetHeight())
+			proof, err := suite.Z().QueryChannelProof(suite.Z().Chain.LastHeader.GetHeight())
 
-			err := suite.A().Chain.App.GetIBCKeeper().ChannelKeeper.ChanOpenAck(
-				suite.A().Chain.GetContext(), suite.A().ChannelConfig.PortID, suite.A().ChannelID,
-				channelCap, suite.Z().ChannelConfig.Version, suite.Z().ChannelID,
-				proof, suite.A().GetClientState().GetLatestHeight(),
+			if tc.expPass {
+				suite.Require().NoError(err)
+			} else {
```

Comment:
another place we can make the linter happy: `elseif: can replace 'else {if cond {}}' with 'else if cond {}' (gocritic)`

Reply:
Fixed